go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -241,7 +241,8 @@ func runOvnKube(ctx *cli.Context) error {
 		// since we capture some metrics in Start()
 		metrics.RegisterMasterMetrics(ovnNBClient, ovnSBClient)
 
-		ovnController := ovn.NewOvnController(ovnClientset, masterWatchFactory, stopChan, nil, ovnNBClient, ovnSBClient, util.EventRecorder(ovnClientset.KubeClient))
+		gwInterface := ctx.String("egw-interface")
```

Comment:
Shouldn't this be plugged in to the `config` package globals?

Reply:
I will recheck since it's been a while now, but I am almost sure this is how parameters are consumed by master.

Reply:
aand you are right, updating it